sync/class.jetpack-sync-module-updates.php

```diff
@@ -39,6 +49,46 @@ public function init_before_send() {
 		add_filter( 'jetpack_sync_before_send_jetpack_update_themes_change', array( $this, 'expand_themes' ) );
 	}
 
+	public function update_core( $new_wp_version ) {
+		global $pagenow;
+
+		if ( isset( $_GET[ 'action' ] ) && 'do-core-reinstall' == $_GET[ 'action' ] ) {
+			/**
+			 * Sync event that is fires when the core reinstall was successful
```

Comment:
We should rephrase:
> * Sync event that is fires when the core reinstall was successful

"Fired on successful core reinstall"? @gititon could you help us here? :)